src/Maker/AbstractMaker.php

```diff
@@ -22,6 +23,9 @@
  */
 abstract class AbstractMaker implements MakerInterface
 {
+    /** @var array Fully Qualified Classnames for Use Statements */
+    protected $useStatements = [];
```

Comment:
Typhint?

Reply:
we still have support for PHP 7.2 until Symfony 5.4 is EOL

Reply:
Check, ok